blog/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self):
+        return f"{self.username}"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(User,
+                              on_delete=models.CASCADE,
+                              related_name="posts")
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(User,
+                             on_delete=models.CASCADE,
+                             related_name="commentary")
+    post = models.ForeignKey(Post,
+                             on_delete=models.CASCADE,
+                             related_name="commentary")
```

Comment:
The related_name for the ForeignKey field should be a plural form as it is used to create a reverse relation from the User and Post models to the Commentary model. So, it would be more appropriate to use 'commentaries' instead of 'commentary'.